docs/developer/checkout/troubleshooting.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Checkout Troubleshooting
+sidebar_label: Troubleshooting
+---
+
+### Missing Shipping Methods
+
+1. Shipping methods are contextual to channels and need to be enabled in the channel the checkout is created.
+2. Shipping methods can be available only for specific countries, if checkout does not contain shipping country the default country of the channel will be used.
+
+### Variant/Product is not available
+
+Make sure product is available in the channel the checkout is created.
```

Comment:
Product related issues should be redirected to this page: https://docs.saleor.io/developer/products/troubleshooting